Python/pystate.c

```diff
@@ -874,6 +874,12 @@ _PyThreadState_DeleteCurrent(_PyRuntimeState *runtime)
     PyEval_ReleaseLock();
 }
 
+void
+PyThreadState_DeleteCurrent()
```

Comment:
Please add void to prevent accepting any argument:

```suggestion
PyThreadState_DeleteCurrent(void)
```